docs/Project-Configuration/Task_Traces.md

```diff
@@ -27,11 +27,23 @@ The following examples illustrate how to inject the trace/span IDs into the cont
 #### Python
 ```python
 from opentelemetry import trace, context
-from opentelemetry.trace import NonRecordingSpan, SpanContext
-
+from opentelemetry.trace import NonRecordingSpan, SpanContext, TraceFlags
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+from opentelemetry.sdk.trace import TracerProvider
+
+# Task traces in Honeycomb are sent with an "evergreen-agent" service name.
+# We need to send test spans with the same service name since the service name
+# determines the Honeycomb dataset and traces can not span multiple datasets.
```

Comment:
Is this accurate? I thought it could span datasets, but not environments?

Reply:
😅 .
I pushed a change to just make the trace flags change instead.